### ISSUE TITLE
using finta for computing features

### ISSUE THREAD

**Issue by arnaudgelas** [councilofelders/opensignals#30]
I personally use finta to compute features. Why not add such a dependency?

https://pypi.org/project/finta/

**Comment by uuazed** [comment]
I agree, opensignals should use some existing library to compute those features instead of reinventing the wheel. I never heard of finta, but it looks like it could be a good match
